registration/include/pcl/registration/impl/icp.hpp

```diff
@@ -41,6 +41,7 @@
 #ifndef PCL_REGISTRATION_IMPL_ICP_HPP_
 #define PCL_REGISTRATION_IMPL_ICP_HPP_
 
+#include <pcl/registration/icp.h>
```

Comment:
Is it necessary with this one?
Seems to give circular includes, since icp.h includes icp.hpp as well?

But it is of course in #guards statements.

Reply:
Yep, you are right. This won't lead to errors because of the include guards, but is also not necessary, so I removed it.